### PR TITLE
Fixed wrong timezone issue

### PR DESCRIPTION
**Pull request by Uriger**
Merged with latest code and inserted the timezone fixes.

### CODE REVIEW

**Comment by MrChrisW** [comment]
@uriger We'll be sure to review these changes shortly. Thanks again :+1:

File: share/spice/get_events/get_events.js [hunk 1]
```diff
@@ -50,17 +50,29 @@
...
+            //console.log(dates.plain_start.format('H:mm'), dates.plain_end.format('H:mm'))
+            return true;
+        }
+        return false;
+    }
 
     function getStartEnd(s, e) {
-        var start = moment(s),
+        //var start = moment(s)
+        var start = moment(new Date(s)).utc(),
```
Comment (moollaza) [inline share/spice/get_events/get_events.js]: I don't believe you need to create the date object here? Moment accepts strings and turns them into Date objects itself.

**Comment by Uriger** [comment]
Fixed the unnessesary creation of date.

**Comment by jagtalon** [comment]
@Uriger LGTM! :+1:

**Comment by Uriger** [comment]
Hey guys,

Github says the subject has been merged but it seems like the changes are
not yet inside the live duckduckgo search.
Any idea when this will be live or what is the process that is happening
now?

Thanks,
Uri
GetEvents

2015-05-06 18:18 GMT+03:00 Jag Talon <notifications@github.com>:

**Comment by moollaza** [comment]
Hi Uri,

They'll go live with the next deploy,  should be sometime this week.

Cheers,

--
Zaahir Moolla
https://duckduckgo.com

**Pull request merged**